### PR TITLE
Add obstacle status telemetry

### PR DESCRIPTION
**Pull request by UsamaEquinorAFK**

### CODE REVIEW

File: src/isar_robot/robotinterface.py [hunk 1]
```diff
@@ -141,6 +144,21 @@ def get_telemetry_publishers(
...
+            mqtt_queue=queue,
+            telemetry_method=self._get_obstacle_status_telemetry,
+            topic=f"isar/{isar_id}/obstacle_status",
+            interval=1,
+            retain=False,
+        )
+        obstacle_status_thread: Thread = Thread(
+            target=obstacle_status_publisher.run,
+            args=[isar_id, robot_name],
+            name="ISAR Robot obstacle status Publisher",
```
Comment (aestene) [inline src/isar_robot/robotinterface.py:157]:
```suggestion
            name="ISAR Robot Obstacle Status Publisher",
```

File: src/isar_robot/robotinterface.py [hunk 2]
```diff
@@ -298,3 +327,12 @@ def _update_pressure_level(self) -> float:
...
         return self.pressure_level
+
+    def _update_obstacle_status(self) -> str:
+        random_obstacle = random.random()
+        if random_obstacle < 0.75:
+            self.obstacle_status = "{}"
+        else:
+            self.obstacle_status = "{'x': 20, 'y':20 }"
+
+        return self.obstacle_status
```
Comment (aestene) [inline src/isar_robot/robotinterface.py:338]: Should not report obstacles

File: src/isar_robot/robotinterface.py [hunk 3]
```diff
@@ -187,6 +205,17 @@ def _get_battery_telemetry(self, isar_id: str, robot_name: str) -> str:
         )
         return json.dumps(battery_payload, cls=EnhancedJSONEncoder)
 
+    def _get_obstacle_status_telemetry(self, isar_id, robot_name: str) -> str:
```
Comment (aestene) [inline src/isar_robot/robotinterface.py:208]:
```suggestion
    def _get_obstacle_status_telemetry(self, isar_id: str, robot_name: str) -> str:
```

File: src/isar_robot/robotinterface.py (hunk 1 above)
Comment (UsamaEquinorAFK) [reply src/isar_robot/robotinterface.py:157]: Done

File: src/isar_robot/robotinterface.py (hunk 2 above)
Comment (UsamaEquinorAFK) [reply src/isar_robot/robotinterface.py]: Done.

**Review by aestene — APPROVED** [review]
LGTM

**Pull request merged**